io.github.baldeuniversel.am-okay.yml

```diff
@@ -0,0 +1,76 @@
+id: io.github.baldeuniversel.am-okay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk 
+
+command: am-okay
+
+finish-args:
+    - --filesystem=host
+    - --filesystem=/tmp
+    - --device=all
+    - --filesystem=/media
+
+modules:
+    - name: am-okay-package
+      buildsystem: simple      
+      sources:
+          - type: git 
+            url: https://github.com/baldeuniversel/am-okay.git 
+            commit: bc41c3ea3124fdc97460bde483b42d1dde66646c 
+
+      build-commands: 
+            - install -D am-okay/bin/am-okay -t /app/bin/am-okay 
+
+            - |
+              install -D am-okay/share/applications/io.github.baldeuniversel.am-okay.desktop -t \
+              /app/share/applications 
+
+            - |
+              install -D am-okay/share/metainfo/io.github.baldeuniversel.am-okay.metainfo.xml -t \
+              /app/share/metainfo
+            
+            - |
+              install -D am-okay/share/icons/hicolor/128x128/apps/io.github.baldeuniversel.am-okay.png -t \
+              /app/share/icons/hicolor/128x128/apps
+
+            - |
+              install -D am-okay/usr/lib/am-okay/library/am-okay-reset-actions.sh \
+              /app/usr/lib/am-okay/library/am-okay-reset-actions.sh
+
+            - |
+              install -D am-okay/usr/lib/am-okay/library/progress-bar-array-copy.sh \
+              /app/usr/lib/am-okay/library/progress-bar-array-copy.sh
```

Comment:
/app/usr cannot exist, also copy them all at once instead of individually installing them

Same for below

Reply:
`/app/usr`  cannot exist ? 

The program works properly .

Reply:
So, I'm going to remove `usr` .